src/v0/destinations/salesforce/utils.js

```diff
@@ -77,15 +100,19 @@ const salesforceResponseHandler = (destResponse, sourceMessage, authKey, authori
     } else if (status === 503 || status === 500) {
       // The salesforce server is unavailable to handle the request. Typically this occurs if the server is down
       // for maintenance or is currently overloaded.
-      throw new RetryableError(
-        `${DESTINATION} Request Failed - due to "${
-          response && Array.isArray(response) && response[0]?.message?.length > 0
-            ? response[0].message
-            : JSON.stringify(response)
-        }", (Retryable) ${sourceMessage}`,
-        500,
-        destResponse,
-      );
+      // ref : https://help.salesforce.com/s/articleView?id=000387190&type=1
+      if (matchErrorCode('SERVER_UNAVAILABLE')) {
+        throw new ThrottledError(
+          `${DESTINATION} Request Failed: ${status} - due to Search unavailable, ${sourceMessage}`,
```

Comment:
Can't we use `getErrorMessage` here too - helps if there are any other error types that can come under SERVER_UNAVAILABLE code

Reply:
edited